subprojects/gradle/cd/src/gradleTest/kotlin/com/avito/ci/steps/LintReportTest.kt

```diff
@@ -0,0 +1,80 @@
+package com.avito.ci.steps
+
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.TestResult
+import com.avito.test.gradle.gradlew
+import com.avito.test.gradle.module.AndroidAppModule
+import com.avito.test.gradle.plugin.plugins
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+import java.nio.file.Path
+
+internal class LintReportTest {
+
+    private lateinit var projectDir: File
+
+    @BeforeEach
+    fun setup(@TempDir tempPath: Path) {
+        projectDir = tempPath.toFile()
+    }
+
+    @Test
+    fun `lint check - does not send error report - valid project`() {
+        generateProject()
+
+        val buildResult = runBuild()
+        buildResult.assertThat()
+            .buildSuccessful()
+            .taskWithOutcome(":app:lintRelease", TaskOutcome.SUCCESS)
+            .taskWithOutcome(":app:lintReportToChannel", TaskOutcome.SUCCESS)
+    }
```

Comment:
This test reproduces the problem.

I didn't test a transport in case or failed lint. 
Faced currently with some restrictions:
- integTests don't support Gradle test kit
- Can't run integTests with slack locally

I'll file a separate issue to support these scenarios.